api-reference/50 Common/Object Structures/globalConfig/defaultUseCurrencyAccountingStyle.md

```diff
@@ -5,7 +5,19 @@ default: true
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies whether to apply the accounting style to formatted numbers of the `currency` [type](/Documentation/ApiReference/Common/Object_Structures/format/#type).
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+The accounting style adds parentheses to negative numbers instead of the minus sign. To disable the accounting style globally and display formatted numbers of the `currency` [type](/Documentation/ApiReference/Common/Object_Structures/format/#type) with the minus sign, set this property to **false**.
+
+    DevExpress.config({ defaultUseCurrencyAccountingStyle: false });
+    // ===== or when using modules =====
+    import config from "devextreme/core/config";
+        
+    config({
+        defaultUseCurrencyAccountingStyle: false 
+    });
+
+This property specifies the global configuration. If you want to override this setting in the component, specify the [useCurrencyAccountingStyle](/Documentation/ApiReference/Common/Object_Structures/format/#useCurrencyAccountingStyle) property in the [format](/Documentation/ApiReference/Common/Object_Structures/format/) object.
```

Comment:
In **any** component, am I right? As for the `format` object, to which component it belongs? Does every component has its own `format` object? Right now it's not quite clear.

```suggestion
This property specifies the global configuration. If you want to override this setting in a component, specify the [useCurrencyAccountingStyle](/Documentation/ApiReference/Common/Object_Structures/format/#useCurrencyAccountingStyle) property in the [format](/Documentation/ApiReference/Common/Object_Structures/format/) object.
```

Reply:
Some of the components have the `format` property, which has a common (for all the components) data type also called `format`. The `format` can be declared as a string, object, or a function. Basically I tried to tell users something like this: if your component value can be formatted, i.e. has the `format` property in its configuration, you can specify the useCurrencyAccountingStyle option using specifically an object as a declaration for the `format` property.

Reply:
OK, I see now. That means it should be something like that:

```suggestion
This property specifies the global accounting style setting. If you want to override this setting for a component, specify the [useCurrencyAccountingStyle](/Documentation/ApiReference/Common/Object_Structures/format/#useCurrencyAccountingStyle) property in the component's [format](/Documentation/ApiReference/Common/Object_Structures/format/) object.
```